zkevm-circuits/src/pi_circuit.rs

```diff
@@ -404,7 +406,9 @@ impl<F: Field, const MAX_TXS: usize, const MAX_CALLDATA: usize>
         offset += 1;
 
         // coinbase
-        let coinbase = block_values.coinbase.to_scalar().unwrap();
+        let mut coinbase_bytes = [0u8; 32];
+        coinbase_bytes[12..].clone_from_slice(block_values.coinbase.as_bytes());
+        let coinbase = rlc(coinbase_bytes, randomness);
```

Comment:
I think this was correct, we should instead update 892 to be `result[offset] = block_values.coinbase.to_scalar().unwrap()`
```suggestion
```